ExtremeValueProject/WeakConvergenceCDF.lean

```diff
@@ -57,7 +103,17 @@ lemma CumulativeDistributionFunction.integral_sum_indicator_eq (F : CumulativeDi
   -- It may be worthwhile to think about an improved phrasing of this.
   -- The previous lemma `CumulativeDistributionFunction.integral_indicator_eq` should be
   -- the key anyway.
-  sorry -- **Issue #18**
+  have h_int_sum_change : ∫ (x : ℝ), (∑ j ∈ s, (Ioc (as j) (bs j)).indicator (fun x => α j)) x ∂F.measure  = ∑ j ∈ s, ∫ (x : ℝ), (Ioc (as j) (bs j)).indicator (fun x => α j) x ∂F.measure  := by
+    rw [← MeasureTheory.integral_finset_sum]
+    simp_all only [measurableSet_Ioc, implies_true, Finset.sum_apply, MeasureTheory.integral_indicator_const]
+    intro j _
+    exact (MeasureTheory.integrable_const (α j)).indicator measurableSet_Ioc
+  rw [h_int_sum_change]
+  congr
+  ext j
+  apply CumulativeDistributionFunction.integral_indicator_eq
+  exact s
+  exact h j
```

Comment:
Sorry --- I had accidentally left some completely irrelevant arguments in the previous lemma statements, which made this part way messire than it needed to be. With the fixes above, the following should work:

```suggestion
  exact F.integral_indicator_eq (h j) _
```